.github/workflows/publish.yml

```diff
@@ -22,3 +22,18 @@ jobs:
       - run: npm whoami; npm publish
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+  publish-registry:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+          cache: npm
+      - run: npm ci
+      - run: npm test
+      - run: npm version ${TAG_NAME} --git-tag-version=false
+        env:
+          TAG_NAME: ${{ github.event.release.tag_name }}
+      - run: npm whoami; npm publish --ignore-scripts --@github:registry='https://npm.pkg.github.com'
```

Comment:
you probably don't want `--ignore-scripts` here, now.

Reply:
good catch! removed